ddprof-ffi/src/exporter.rs

```diff
@@ -221,7 +221,7 @@ fn try_to_url(slice: ByteSlice) -> Result<hyper::Uri, Box<dyn std::error::Error>
     let str: &str = slice.try_into()?;
     #[cfg(unix)]
     if let Some(path) = str.strip_prefix("unix://") {
-        return Ok(ddprof_exporter::socket_path_to_uri(path.as_ref())?);
+        return ddprof_exporter::socket_path_to_uri(path.as_ref());
```

Comment:
I'm curious -- why is the `Ok` not needed anymore? Is it assumed as the default on a `return` if you don't state it anywhere?

Reply:
`socket_path_to_uri` already returns a `Result` -- there's no need to use `?` and wrap it back into `Ok` ^_^